### DISCUSSION TITLE
Update for Craft 5 compatibility

### DISCUSSION THREAD

**Discussion by pedem** [trendyminds/algolia#17] [Q&A] [answered]
Is there a timeline for Craft 5 compatibility?  I know that last time this question was asked for Craft 4 compatibility, you guys were on it in a matter of days.  I don't expect that to be the case, but you've set the precedent of being very responsive!

**Comment by aaronbushnell** [comment]
Thanks, @pedem! We sometimes wait a bit to see if there's any demand for these migrations just in the case folks aren't really using these plugins any longer!

I'd guess we'll have this migrated by early EOW, stay tuned! ❤️

**Reply by aaronbushnell** [reply, marked as answer]
Apparently I had already done most of the work and just forgot about it! Pushed out the Craft 5 version today, @pedem!
